php-frontend/src/test/java/org/sonar/php/highlighter/SymbolHighlighterTest.java

```diff
@@ -77,17 +77,17 @@ public void setUp() throws IOException {
   public void test_empty_input() throws Exception {
     highlight("<?php ");
 
-    checkSymbolExistence(1, 1, false);
+    checkNoSymbolExists(1, 1);
   }
 
   @Test
   public void test_no_usages() throws Exception {
     highlight("<?php   $a = 1; ");
 
-    checkSymbolExistence(1, 7, false);   // (blank)
```

Comment:
yes. I fixed some tests at the same time, that's why the commit is "Update headers and fix tests"